internal/node/node.bzl

```diff
@@ -361,6 +362,7 @@ fi
         # since this will be called from a nodejs_test, where the entrypoint is going to be the test file
         # we shouldn't add the entrypoint as a attribute to collect here
         coverage_common.instrumented_files_info(ctx, dependency_attributes = ["data"], extensions = ["js", "ts"]),
+        RunEnvironmentInfo(expanded_env),
```

Comment:
This means it doesn't get the right env when run directly in `bazel-bin` right? Or when run as a data dependency in the runfiles of something else?